src/UI.rs

```diff
@@ -155,6 +156,26 @@ pub fn ui(f: &mut Frame, app: &App) {
                 ),
         );
 
+    let chart_f = Chart::new(dataset)
```

Comment:
ah this is a very interesting solution. instead of having to do the conversion we can just swap the axis labels! smart!